doc/devices/aer.rst

```diff
@@ -33,18 +33,12 @@ You can then execute the circuit like any other function to get the quantum mech
 
     circuit(0.2, 0.1, 0.3)
 
-Backends
-~~~~~~~~
+Backend Methods and Options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I think this section is missing an example with the recommended interface (an initialized backend) - the main example is using as string for the backend